beacon-chain/endtoend/evaluators.go

```diff
@@ -0,0 +1,144 @@
+package endtoend
```

Comment:
Please break up evaluators into their own files, otherwise this file is going to grow very large over time.